### PR TITLE
Updated to work with Ansible 2.0

### LINKED ISSUE #50: Incompatible with Ansible 2.0
Ansible 2.0 just dropped this week.  With it, I get this error:

```
[$]> ansible-shell                                                                                                                                                                                              
Traceback (most recent call last):
  File "/usr/bin/ansible-shell", line 36, in <module>
    import ansible.runner
ImportError: No module named runner
```

I don't see any mention of a re-org of `runner` in [the changelog](https://github.com/ansible/ansible/blob/devel/CHANGELOG.md), but it has definitely moved (or disappeared).

### PR DESCRIPTION
**Pull request by ghjm**
Fixes #50 by switching to the Ansible 2.0 Python API.

Note that this does not work against 1.x any more.

I also added 'ls' as an alias for 'list' because I just keep typing ls.

### CODE REVIEW

**Comment by dominis** [comment]
Many-many thanks for this :raised_hands: 

I'll run some tests tomorrow and try to merge asap. Thank you again for your work!

File: ansible-shell [hunk 1]
```diff
@@ -339,23 +422,15 @@ class AnsibleShell(cmd.Cmd):
...
-        if self.options.cwd == '*':
-            completions = self.hosts + self.groups
+        if self.cli.options.cwd == '*':
+            completions = self.cli.hosts + self.groups
         else:
-            completions = self.ansible.inventory.list_hosts(self.options.cwd)
-
-        return [s[offs:] for s in completions if s.startswith(mline)]
-
-    def completedefault(self, text, line, begidx, endidx):
```
Comment (dominis) [inline ansible-shell]: this shouldn't be removed because this provides the tab completion for the module parameters

Comment (ghjm) [inline ansible-shell]: Yep, I misunderstood that to be getting the base completion of the command (module) name. Will fix.

**Comment by ghjm** [comment]
Let me know if you want me to squash these commits for a cleaner PR.

**Comment by tamsky** [comment]
Ping?

**Comment by dominis** [comment]
https://github.com/ansible/ansible/pull/14824

**Pull request closed without merging**